### PR TITLE
Converted provision script to PHP.

### PR DESCRIPTION
**Pull request by AlexSkrypnyk**
Closes #

## Summary by CodeRabbit

* **New Features**
  * Added a comprehensive site provisioning system with DB import, profile-based install, DB sanitization, maintenance mode, and custom post-provision scripts.

* **Tests**
  * Added extensive unit test coverage for provisioning workflows, edge cases, and post-provision operations.

* **Documentation**
  * Added tooling subsystem docs and updated system map and test/run examples to include the new tooling subsystem.

### CODE REVIEW

**Comment by AlexSkrypnyk** [comment]
```
Code Coverage Report:
  2026-01-07 07:36:30

 Summary:
  Classes:  0.00% (0/1)
  Methods:  0.00% (0/2)
  Lines:   93.82% (167/178)
```

**Comment by AlexSkrypnyk** [comment]
```
Code Coverage Report:
  2026-02-20 02:05:37

 Summary:
  Classes:  0.00% (0/1)
  Methods:  0.00% (0/2)
  Lines:   94.65% (177/187)
```

**Comment by AlexSkrypnyk** [comment]
```
Code Coverage Report:
  2026-02-20 02:08:59

 Summary:
  Classes:  0.00% (0/1)
  Methods:  0.00% (0/2)
  Lines:   94.65% (177/187)
```

**Comment by AlexSkrypnyk** [comment]
```
Code Coverage Report:
  2026-02-20 03:17:31

 Summary:
  Classes:  0.00% (0/1)
  Methods:  0.00% (0/2)
  Lines:   94.65% (177/187)
```

File: .vortex/tooling/phpcs.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    <file>check-no-exit.php</file>
+    <file>src</file>
+    <file>src/notify</file>
+    <file>src/notify-email</file>
+    <file>src/notify-github</file>
+    <file>src/notify-jira</file>
+    <file>src/notify-newrelic</file>
+    <file>src/notify-slack</file>
+    <file>src/notify-webhook</file>
```
[the remark this answers is not in the document]
Comment (AlexSkrypnyk) [reply .vortex/tooling/phpcs.xml]: phpcs does not support files without extension

**Comment by AlexSkrypnyk** [comment]
```
Code Coverage Report:
  2026-02-20 09:13:49

 Summary:
  Classes:  0.00% (0/1)
  Methods:  0.00% (0/2)
  Lines:   94.65% (177/187)
```

**Comment by AlexSkrypnyk** [comment]
```
Code Coverage Report:
  2026-02-20 09:16:56

 Summary:
  Classes:  0.00% (0/1)
  Methods:  0.00% (0/2)
  Lines:   94.65% (177/187)
```
